src/installer_core.py

```diff
@@ -184,9 +184,12 @@ def post_bootstrap(super_group):
     os.system("echo {\\'name\\': \\'root\\', \\'children\\': [{\\'name\\': \\'0\\'}]} | sudo tee /mnt/.snapshots/ash/fstree") # Initialize fstree
   # Create user and set password
     set_password("root")
-    username = get_username()
-    create_user(username, super_group)
-    set_password(username)
+    if distro !="kicksecure":
+        username = get_username()
+        create_user(username, super_group)
+        set_password(username)
+    else:
+        print("Username is 'user' please change the default password")
```

Comment:
Thanks for the PR. All good on other changes, but is there a better more generic way to handle this? I would like to not mention specific distro in installer_core.py (as much as possible).

Reply:
It will work if you use the old code, but you will have two users, one with weak password and the other needs to be in console + sudo groups to work properly.
I tried to remove the default user and replace it with a new one, but I had some error messages which needs more work and time that I don't have now, but maybe in the future.

Reply:
I'll merge this to upstream, but please feel free to submit any new improvements.